# Generated by Django 5.2.3 on 2025-07-09 07:18

from django.conf import settings
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('food', '0004_alter_favorite_options_and_more'),
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
    ]

    operations = [
        migrations.RenameModel(
            old_name='Favorite',
            new_name='Favorites',
        ),
        migrations.RemoveConstraint(
            model_name='favorites',
            name='unique_addition_in_favorite',
        ),
        migrations.AddConstraint(
            model_name='favorites',
            constraint=models.UniqueConstraint(fields=('user', 'recipe'), name='unique_addition_in_favorites', violation_error_message='Рецепт уже добавлен.'),
        ),
    ]
